templates/features.html

```diff
@@ -94,6 +94,12 @@
   search.value = location.hash.substr(1);
 }
 
+chromeMetadata.addEventListener('queryChanged', function(e) {
+  search.value = '=' + e.detail.version;
+  var event = new Event('input');
+  search.dispatchEvent(event);
```

Comment:
dispatching a synthetic input event seems like a bit of a hack.  I'd personally just have the two listeners call into a common "update search" function or something.